### ISSUE TITLE
Add Patches and system-apps

### ISSUE THREAD

**Issue by rkunschke** [hashbang/aosp-build#14]
Hello,
I try'ed to build aosp with microg so I added the packages in the config/manifest/base.xml as it is descriped here: https://github.com/microg/android_packages_apps_GmsCore/wiki/Building
I used make DEVICE=sargo fetch to get the sources and jumped into the container with make shell. I applyed this patch without problems. https://github.com/microg/android_packages_apps_GmsCore/blob/master/patches/android_frameworks_base-P.patch

I started the build with make DEVICE=sargo and the build failed because it cant write the new apps to the desired folder.

I don't think what I used the build-system how it should be used.

How can i add system-apps and apply patches?

**Comment by ypid** [comment]
I have not done this myself and have not intention to do so. But as I understand it, you would need to follow this part of the documentation: https://github.com/microg/android_packages_apps_GmsCore/wiki/Building#integrate-gmscore-in-aosp-based-rom Did you do that?

**Comment by rkunschke** [comment]
Yes. But I am not sure how to do it. Should i do it in the aosp-build folder or in the shell with make shell?

**Comment by ypid** [comment]
`make shell` would be preferred. Please get familiar with Docker and with the layout of this project.
